library/include/rocblas_bfloat16.h

```diff
@@ -57,8 +57,8 @@ struct rocblas_bfloat16
     __host__ __device__ rocblas_bfloat16() = default;
 
     // round upper 16 bits of IEEE float to convert to bfloat16
-    explicit constexpr __host__ __device__ rocblas_bfloat16(float f)
-        : data(float_to_bfloat16(f))
+    explicit constexpr __host__ __device__ rocblas_bfloat16(float f, bool round = true)
+        : data(round ? float_to_bfloat16(f) : truncate(f))
```

Comment:
It looks like the first function takes in round, is this ternary still needed and the truncate function needed?

Reply:
Oops, you're right. I tried both versions, seeing which looked cleaner and forgot I left both in. Probably tidiest to have the rounding option in float_to_bfloat16 rather than the separate truncate function I think?